src/components/GymLogo/GymLogo.tsx

```diff
@@ -0,0 +1,50 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import { color, typeface } from '../../styles';
+import gymLogo from '../../images/gymnasium-logo-white.svg';
+
+interface Props {
+  /**
+   * Logo width override
+   */
+  width?: string;
+  /**
+   * Logo height override
+   */
+  height?: string;
+  /**
+   * Background color override
+   */
+  backgroundColor?: string;
+}
+
+/**
+ * `GymLogo` is the standard Gymnasium logo
+ */
+const GymLogo: React.FunctionComponent<Props> = ({
+  width = '200px',
+  height = '23px',
+  backgroundColor = color.grey.dark,
```

Comment:
Ah, one thing - This should default to transparent - you can use the already configured [backgrounds addon](https://github.com/storybookjs/storybook/tree/master/addons/backgrounds) in storybook to set up the background color for stories if that's helpful.